.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: dart
+
+# Workaround for issue with sandboxed Chrome in containerized Travis builds.
+sudo: required
+addons:
+  chrome: stable
+
+# Re-use downloaded pub packages everywhere.
+cache:
+  directories:
+  - $HOME/.pub-cache
+  - $HOME/.dart_tool
+
+jobs:
+  include:
+    - stage: Dart 2
+      dart: stable
+      script:
+        - dartanalyzer .
+        - pub run dependency_validator -i build_runner,build_test,build_web_compilers
+        - pub run test -p chrome
+        - pub run build_runner test -- -p chrome
```

Comment:
So this runs tests in DDC and the previous test runs in dart2js, correct?

Reply:
Ah, yup, the README confirms this. Nice 😄

Reply:
correct @greglittlefield-wf